### DISCUSSION TITLE
Feeds stopping and starting even with huge vide/audio buffers

### DISCUSSION THREAD

**Discussion by tonogdlp** [f1multiviewer/issue-tracker#32] [Q&A] [answered]
Hi! first I wanted to thank you for this awesome app! I know it is in beta, but it already beats any other way of watching F1 at the moment. 

However, I can watch one or two (maybe three) at the same time with no problems, but anytime I go for more than 4 feeds they start to become unwatchable. Happened both on a live feed, and in replays. They start pausing every 5 seconds or so. They pause for about 1 second, then keep playing. It happens in all of the open feeds, and not at the same time. 

Currently running beta 5, on 2018 MacBook Pro, macOS 12.4 

Most feeds have a debug screen something like this:
![Screen Shot 2022-06-11 at 12 17 36](https://user-images.githubusercontent.com/8254361/173202024-17af584d-bd9a-4da3-b9a8-0c8659a34a6a.png)

I have 60 seconds of video and audio already in my computer, so I know that it is not an Internet problem. I have tested this by shutting off my WiFi, and keep playing each feed. All still continue without a hiccup (individually)

iStats shows this:
![Screen Shot 2022-06-11 at 12 26 24](https://user-images.githubusercontent.com/8254361/173202165-7195573e-cc32-4597-a3b4-bd0972ed3892.png)

Other things I tried:
Opened 3 feeds, max video quality in all. No problem. 
Opened 4 feeds, min video quality in all. They start pausing.

**Comment by f1multiviewer** [comment, marked as answer]
From your screenshot, it seems like your VRAM is completely full. Currently, quality selection doesn't work, so maybe your machine cannot handle this many full-quality streams. I'm not sure what your uptime is, but macOS isn't known for it's great video memory management, so maybe rebooting may help relief your memory constraint. If not, I hope that the future update with full support for manual quality selection works better.

**Reply by tonogdlp** [reply]
Thank you for your answer! I'll try that next time!
